README.md

```diff
@@ -1,2 +1,115 @@
-# python-common
+# shipchain-common
```

Comment:
Since the readme title is what's displayed at the top of the PyPi page, I think the name should match the installable package.  Unless we want to make it neither and title it `Shipchain Common Python Library`

![image](https://user-images.githubusercontent.com/38572212/70242231-76658f80-173e-11ea-8925-5e097350ee8e.png)

Reply:
Actually, I think I prefer that, having `shipchain-common` on that page 3x is a little redundant anyways